apps/api/src/simulation-run/simulation-run.controller.ts

```diff
@@ -217,33 +219,38 @@ export class SimulationRunController {
       envVars: run.envVars,
       purpose: run.purpose,
       projectId: projectId,
-      projectOwner: owner?.sub,
+      projectOwner: user?.sub,
     };
     const sim = await this.dispatchQueue.add(message);
 
     return response;
   }
 
+  private checkPublishProjectPermission(user: AuthToken, projectId?: string): void {
+    if (
+      projectId
+      && !user?.permissions?.includes(scopes.simulationRuns.externallyValidate.id) // TODO: debug for machine accounts
```

Comment:
@bilalshaikh42 This is the point where machine account information is no longer available

Reply:
I am not able to replicate this. On #3628 I see an issue with the code you are using. Does changing that fix it?